main.py

```
@@ -146,6 +153,13 @@ def _schedule_initialization_tasks(self):
         Clock.schedule_once(self.startup_update_check, 3)
     
     def open_detail(self, sensor_key: str, screen_name: str):
+            """
```

Flake8 (7.2.0)

[error] over-indented

(E117)